### ISSUE TITLE
When Downloading fa tNotes from fa_gl, tCore gets stuck in a loop due to missing resources.

### ISSUE THREAD

**Issue by PhotoNomad0** [unfoldingWord/translationCore#7498]
### User Story
When you try to download fa_gl/fa_ta in tc, it gets stuck in a loop trying to download dependency https://git.door43.org/fa_gl/fa_ta/archive/v69.zip which it can't find. 

tC should give up and show error if dependent resource does not exist.

**Additional data:**
Correction - the fa_ta does exist, but the manifest and data are actually tNotes.  Still we need to fail more gracefully. @Rich Mahn 

### Features / Specifications

- [ ]
- [ ]
- [ ]

### Definition of Done

- [ ]
- [ ]
- [ ]

**Comment by RUN-Projects** [comment]
It seems to be happy now if all fa resources except for tN are added. It seems to be just tN that it doesn't like (at least on linux).

**Comment by RUN-Projects** [comment]
Should this be 2 different issues -- one to sort out fa tN, and the other to add something helpful to the whitescreen (like which resource directory to delete?)?

**Comment by elsylambert** [comment]
@PhotoNomad0  I was on tC 3.6.2 on Mac and was able to download fa_gl bibles and translationHelps resources. The translationNotes folder contains only this manifest file and no other content.
This is the manifest file:  
[manifest.json](https://github.com/unfoldingWord/translationCore/files/13891835/manifest.json)
**_Log file:_** 
[Wed Jan 10 2024.log.zip](https://github.com/unfoldingWord/translationCore/files/13891838/Wed.Jan.10.2024.log.zip)

This is the link from DCS for ga_tn: https://git.door43.org/fa_gl/fa_tn and not seeing all these tsv files in tC.
Door43 Content Service
[fa_tn](https://git.door43.org/fa_gl/fa_tn)

**Comment by PhotoNomad0** [comment]
@elsylambert Looking in the fa_tn manifest.yaml, it shows all the books of the Bible the the projects list: https://git.door43.org/fa_gl/fa_tn/src/branch/master/manifest.yaml .  But in the repo itself, only 8 books are present: https://git.door43.org/fa_gl/fa_tn

**Comment by PhotoNomad0** [comment]
In the fa_glt the same issue (https://git.door43.org/fa_gl/fa_glt/src/branch/master/manifest.yaml) shows all the books of the Bible the the projects list, but only 8 books are present in the repo: https://git.door43.org/fa_gl/fa_glt

**Comment by PhotoNomad0** [comment]
@elsylambert I see the same problem with just the manifest in the folder.  Trying again after deleting folder.

**Comment by PhotoNomad0** [comment]
@elsylambert @birchamp It looks like the problem is this field in the manifest:
```
    "relation": [
      "fa/ult",
      "fa/ust",
      "hbo/uhb",
      "el-x-koine/ugnt",
      "fa/ta",
      "fa/tq",
      "fa/tw"
    ],
```

TN needs to know what version of the original languages where used in making the selections, so it can load that specific version from Door43.  It should be something like:
```
    "relation": [
      "en/ult?v=41",
      "en/ust?v=40",
      "hbo/uhb?v=2.1.30",
      "el-x-koine/ugnt?v=0.30",
      "en/ta?v=33",
      "en/tq?v=38",
      "en/tw?v=36"
    ],
```

**Comment by PhotoNomad0** [comment]
@elsylambert @birchamp - It looks like the manifest has already been fixed in the master branch (https://git.door43.org/fa_gl/fa_tn/src/branch/master/manifest.yaml).  They just need to make a new release with the fix.

**Comment by elsylambert** [comment]
@birchamp  It looks like the manifest has already been fixed in the master branch (https://git.door43.org/fa_gl/fa_tn/src/branch/master/manifest.yaml).  They just need to make a new release with the fix.

**Comment by PhotoNomad0** [comment]
Fix is in translationCore 3.6.3 (e859026)

**Comment by elsylambert** [comment]
translationCore 3.6.3 (e859026). tC shows error on fa_tn download as in the screenshot:
<img width="1602" alt="Screenshot 2024-03-11 at 4 35 41 PM" src="https://github.com/unfoldingWord/translationCore/assets/51806180/d9ca9c7a-774c-4662-9186-8f259cf501db">
